lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
Yes you'll need to mimic citibike. Some of the files will be in the same directory.

Take a look at the Transit::Septa, Transit::Swirtzerland to see how they're organized.

There's a standard naming convention: the filenames and paths are based off of the Perl package name. We take it from `CamelCase` to `snake_case`.

If you run `duckpan new` and give the package name it'll create all the files for you :)

![1____projects_zeroclickinfo-spice__bash_](https://cloud.githubusercontent.com/assets/873785/9721048/d36cbe58-5562-11e5-9b97-63be443c81a4.png)